### DISCUSSION TITLE
What is this error? (ValidationProvider)

### DISCUSSION THREAD

**Discussion by meteoguzhan** [adonisjs/core#4001] [Help]
Hello,
command: node ace generate:manifest

Invalid command "ValidationProvider". Make sure to define the static property "commandName"

   1  validateCommand
     /project/node_modules/@adonisjs/ace/build/src/utils/validateCommand.js:25

   2  ManifestGenerator.loadCommand
     /project/node_modules/@adonisjs/ace/build/src/Manifest/Generator.js:53

   3  ManifestGenerator.loadCommands
     /project/node_modules/@adonisjs/ace/build/src/Manifest/Generator.js:67

   4  ManifestGenerator.generate
     /project/node_modules/@adonisjs/ace/build/src/Manifest/Generator.js:76

   5  GenerateManifest.handle
     /project/node_modules/@adonisjs/core/build/src/Ignitor/Ace/GenerateManifest/index.js:67

   6  Ace.handle
     /project/node_modules/@adonisjs/core/build/src/Ignitor/Ace/index.js:27

**Comment by e-e** [comment]
Check to make sure all of your commands have the `commandName` static property set:

Example:
```ts
export default class ExampleCommand extends BaseCommand {
  /**
   * <THIS NEEDS TO BE SET>
   */
  public static commandName = "example:command";

  // ... rest of the command class

}
```

the value can be whatever you want as long as it is unique with respect to your other commands

**Reply by meteoguzhan** [reply]
> the value can be whatever you want as long as it is unique with respect to your other commands

Thank you for your answer but I don't have any commands.
